frontend/ui/lib/pages/trip_view_page.dart

```diff
@@ -1,13 +1,23 @@
 import 'package:flutter/material.dart';
+import 'package:tripmeout/services/trip_service.dart';
 import 'package:tripmeout/widgets/trip_view_widget.dart';
 import 'package:tripmeout/widgets/default_app_bar.dart';
 
 class TripViewPage extends StatelessWidget {
+  final TripService tripService;
+  final String tripId;
+
+  TripViewPage(this.tripService, this.tripId, {Key key}) : super(key: key);
+
   @override
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: defaultAppBar(context),
-      body: Center(child: TripViewWidget()),
+<<<<<<< HEAD
```

Comment:
Looks like this was accidentally committed with the merge conflict markers. These need to be fixed.